pulsar-client-admin/src/main/java/org/apache/pulsar/client/admin/Lookup.java

```diff
@@ -41,6 +42,14 @@
      */
     CompletableFuture<String> lookupTopicAsync(String topic);
 
+    /**
+     * Lookup a partitioned topic.
+     *
+     * @param topic
+     * @return the broker URL that serves the topic
+     */
+    Map<String, String> lookupPartitionedTopic(String topic) throws PulsarAdminException;
```

Comment:
This method should return a CompletableFuture as the method above

Reply:
ditto